cypress/integration/smoke/balances.spec.js

```diff
@@ -0,0 +1,196 @@
+const assetsTableContainer = '.MuiTableContainer-root'
+const balanceRowTestId = '[data-testid=balance-row]'
+const receiveModalClass = '.receive-modal'
+
+const TEST_SAFE = '/rin:0x11Df0fa87b30080d59eba632570f620e37f2a8f7'
+const ASSETS_LENGTH = 7
+
+describe('Assets > Coins', () => {
+  const TOKEN_AMOUNT_COLUMN = 1
+  const FIAT_AMOUNT_COLUMN = 2
+
+  // Fiat balance regex
+  const fiatRegex = new RegExp(`([0-9]{1,3},)*[0-9]{1,3}.[0-9]{2}`)
+  // Token balance regex
+  const tokenRegex = new RegExp(`([0-9]{1,3},)*[0-9]{1,3}.[0-9]{2,5}`)
```

Comment:
I don't think these are region agnostic?

Reply:
@katspaugh suggested that we'd do literal assertions for the `thousand` and `decimal` separators. Please see https://github.com/safe-global/safe-react/pull/3920#discussion_r893219929

Reply:
Oversaw this. I'll resolve this.